include/nanoflann.hpp

```diff
@@ -831,16 +838,16 @@ class KDTreeBaseClass {
   size_t usedMemory(Derived &obj) {
     return obj.pool.usedMemory + obj.pool.wastedMemory +
            obj.dataset.kdtree_get_point_count() *
-               sizeof(IndexType); // pool memory and vind array memory
+               sizeof(AccessorType); // pool memory and vind array memory
   }
 
-  void computeMinMax(const Derived &obj, IndexType *ind, IndexType count,
+  void computeMinMax(const Derived &obj, uint64_t ind, uint64_t count,
```

Comment:
Shouldn't this uint64_t be AccessorType too? Or if it's not, another new symbolic name? 
(I'm doing a quick review, sorry if the answer is obvious)

Reply:
To my understanding, what was done here is that a pointer to `IndexType`  was used in order to be able to use the element to which `ind` points to, which is located in `vInd` (or now `vAcc`) as the element 0 of an array. However as pointer arithmetic is quite unsafe, I thought it would be somewhat safer and more understandable to explicitly access `vAcc` with `ind` as an offset, as we do have access to the array here. By using the `std::vector::operator[]` explicitly, range checks are at least performed in debug mode.